antismash/common/secmet/features/antismash_domain.py

```diff
@@ -63,16 +78,18 @@ def from_biopython(cls: Type[T], bio_feature: SeqFeature, feature: T = None,
 
 
 _SUBTYPE_MAPPING: Dict[str, Type[AntismashDomain]] = {}
+_SUBSUBTYPE_MAPPING: Dict[str, Type[AntismashDomain]] = {}
 
 
-def register_asdomain_variant(tool: str, subtype: Type[AntismashDomain]) -> None:
+def register_asdomain_variant(tool: str, subtype: Type[AntismashDomain], subsubtype: Type[AntismashDomain]="") -> None:
```

Comment:
This is not the right place to make the changes you need. You are instead looking for `ModularDomain` in `antismash/detection/nrps_pks_domains/modular_domain.py` which is specific to NRPS/PKS domains.